README.MD

```diff
@@ -5,7 +5,7 @@
 
 Dependencies:
 [<img src="https://img.shields.io/badge/dynamic/json?color=blue&url=https://raw.githubusercontent.com/eclipse-emfcloud/theia-tree-editor/master/theia-tree-editor/package.json&query=$.dependencies%5B%27%40jsonforms%2Fcore%27%5D&label=JSONForms&logo=JSONForms" alt="JSONForms"/>](https://github.com/eclipsesource/jsonforms)
-[<img src="https://img.shields.io/badge/dynamic/json?color=blue&url=https://raw.githubusercontent.com/eclipse-emfcloud/theia-tree-editor/master/theia-tree-editor/package.json&query=$.dependencies%5B%27%40theia%2Fcore%27%5D&label=Theia&logo=Theia" alt="Theia"/>](https://github.com/eclipse-theia/theia)
+[<img src="https://img.shields.io/badge/dynamic/json?color=blue&url=https://raw.githubusercontent.com/eclipse-emfcloud/theia-tree-editor/master/theia-tree-editor/package.json&query=$.peerDependencies%5B%27%40theia%2Fcore%27%5D&label=Theia&logo=Theia" alt="Theia"/>](https://github.com/eclipse-theia/theia)
```

Comment:
It seems this query does not deliver a result - I get `no result` on the badge. Maybe we can stick with `dependencies` here?

Reply:
The problem it that it queries the master branch. At the moment of this PR, the master branch does not contain the dependency in `peerDependencies`. However, after the PR is merged, the `@theia/core` dependency will no longer exist in `dependencies` but only in `peerDependencies`.

Reply:
Ah alright, thanks clearing that up for me!